actionsystemtest/lib/action_system_test/test_helpers/active_job_setup.rb

```diff
@@ -0,0 +1,25 @@
+module ActionSystemTest
+  module TestHelpers
+    # Active Job setup helper for system testing
+    module ActiveJobSetup
+      # Sets the queue adapter to async and remembers the original adapter
+      # setting.
+      #
+      # This method should be used in the test setup code so that jobs run
+      # when the test runs because the test can't wait to check for pass/fail
+      # if it depends on running jobs.
```

Comment:
I took this helper from the Basecamp code under the assumption that we want to move as much out as possible. The default adapter in test mode is TestAdapter, not Async, but perhaps that is something that applications should handle on their own. I'll remove it.